src/functions/Coverage.ps1

```diff
@@ -543,6 +550,7 @@ function Get-CoverageCommandText {
 
     $reportParentExtentTypes = @(
         [System.Management.Automation.Language.ReturnStatementAst]
+        [System.Management.Automation.Language.ExitStatementAst]
         [System.Management.Automation.Language.ThrowStatementAst]
```

Comment:
Why are we doing this? To make diagnostic log easier to read?

E.g. `throw 123` will get two different breakpoint locations:
- One for `throw 123` AST
- One for `123` AST
but command text will be `throw 123` and look like duplicates. Same for `exit 123` and `return 123`.

Reply:
I don't have an answer, I remember the profiler based CC was a big fight for making it capture just the right amount of stuff in the right places, so it is entirely possible that I output more trace points than needed.